docs/modules/mechanics/actions-triggers.mdx

```diff
@@ -96,23 +123,63 @@ In the future, some features that are currently used in Kits may be transferred
-  | Attribute | Description | Value |
+  | Sub-elements | Description | Value |
   |---|---|---|
-  | `<find />` | The item to find. | [Item Attributes](/docs/modules/gear/items#item-attributes) |
+  | `<find/>` | The item to find in a player's inventory. | [Item Attributes](/docs/modules/gear/items#item-attributes) |
   | `<replace/>` | The new item to replace with. | [Item Attributes](/docs/modules/gear/items#item-attributes) |
+</div>
+
+#### Attributes
+
+<div className="table-container">
+  | Attribute | Description | Value |
+  |---|---|---|
   | `keep-amount` | Player recives the same amount of the new item as they had of the old item. | <span className="badge badge--primary">true/false</span> |
   | `keep-enchants` | Enchantments on the old item will be applied to the new item. | <span className="badge badge--primary">true/false</span> |
   | `ignore-metadata` | Filters which entities to remove. | <span className="badge badge--primary">true/false</span> |
   | `amount` | Match for item stacks that have a certain amount of items in a range. | <span className="badge badge--primary">Range</span> |
 </div>
 
+### Take-Payment
+
+#### Sub-elements
+
+<div className="table-container">
+  | Sub-elements | Description | Value |
+  |---|---|---|
+  | `<payment/>` | <span className="badge badge--secondary" title="Can be either this attribute or a sub-element.">Property</span><span className="badge badge--danger">Required</span>An individual payment. | [Item Attributes](/docs/modules/gear/items#item-attributes) |
+  | `<success-action/>` | <span className="badge badge--secondary" title="Can be either this attribute or a sub-element.">Property</span>The action to trigger upon a successful payment. | <span className="badge badge--primary">Action</span> |
+  | `<fail-action/>` | <span className="badge badge--secondary" title="Can be either this attribute or a sub-element.">Property</span>The action to trigger upon a failed payment. | <span className="badge badge--primary">Action</span> |
+</div>
+
+#### Attributes
+
+<div className="table-container">
+  | Attribute | Description | Value |
+  |---|---|---|
+  | `material` | <span className="badge badge--danger">Required</span>The item to display as an icon. | [Material Name](/docs/reference/items/inventory#material-matchers) |
+  | `price` | The amount of a currency needed to purchase. | <span className="badge badge--primary">Number</span> |
+  | `currency` | The currency needed to purchase. | [Filter](/docs/modules/mechanics/filters) |
+</div>
+
+### Velocity & Teleport Attributes
+<div className="table-container">
+  | Attribute | Description | Value |
+  |---|---|---|
+  | `x` | The X coordinate, measured in east-west. | <span className="badge badge--primary">Number</span> |
+  | `y` | The Y coordinate, measured in up-down. | <span className="badge badge--primary">Number</span> |
+  | `z` | The Z coordinate, measured in north-south. | <span className="badge badge--primary">Number</span> |
```

Comment:
The teleport action supports `yaw` and `pitch` as well. May also be worth noting that these support variables / expressions